packages/vscode-extension/src/constants.ts

```diff
@@ -27,6 +27,7 @@ export enum GlobalKey {
   ShowLocalDebugMessage = "ShowLocalDebugMessage",
   CreateWarnings = "CreateWarnings",
   SampleGalleryLayout = "teamsToolkit:sampleGallery:layout",
+  AutoInstallDependency = "fx-extension.autoInstallDependency",
```

Comment:
It's better to use `teamsToolkit` as prefix instead of `fx-extension` which is confusing.

Reply:
Updated